universal-application-tool-0.0.1/app/views/dev/AwsStorageDevViewStrategy.java

```diff
@@ -58,4 +77,22 @@ public ContainerTag getFileUploadForm(
         .withMethod("post")
         .withAction(request.actionLink());
   }
+
+  @Override
+  public ContainerTag renderFiles(ImmutableList<StoredFile> files) {
+    return table()
+        .with(
+            tbody(
+                each(
+                    files,
+                    file ->
+                        tr(
+                            td(String.valueOf(file.id)),
+                            td(a(file.getName()).withHref(getPresignedURL(file)))))));
+  }
+
+  @Override
+  public String getPresignedURL(StoredFile file) {
```

Comment:
nit: java style guide
```suggestion
  public String getPresignedUrl(StoredFile file) {
```